ApplicationExamples/SevenBus/SevenBus_Base_Case.mo

```diff
@@ -370,6 +380,11 @@ equation
           extent={{-114,-78},{-84,-88}},
           lineColor={28,108,200},
           textString="FTILL5")}),
-    Icon(coordinateSystem(extent={{-280,-200},{280,180}})),
-    Documentation);
-end Network;
+    Icon(coordinateSystem(extent={{-280,-160},{280,220}})),
+    Documentation,
+    experiment(
+      StopTime=2,
+      __Dymola_NumberOfIntervals=5000,
+      Tolerance=1e-06,
+      __Dymola_Algorithm="Dassl"));
```

Comment:
These annotations are superfluous or non-standard. You should rather define `Interval=`